dao/src/main/java/cz.muni.fi.pa165.soccermanager/entity/Match.java

```diff
@@ -17,24 +58,14 @@
     private Team homeTeam;
     @ManyToOne
     private Team awayTeam;
```

Comment:
Add column notation with nullable set to false also to home team and away team please. I would also add blank line between attributes with notation to increase readability